### DISCUSSION TITLE
How Can I Use Tagging in Mecha CMS?

### DISCUSSION THREAD

**Discussion by ghost** [mecha-cms/mecha#149] [Q&A]
Hi Taufik,

How can I use tagging in Mecha CMS? Reference from this page https://mecha-cms.com/store/extension/tag

Can you tell me, which code snippet I must add to which page? Thanks.

**Comment by taufik-nurrohman** [comment]
You can use it right away from the Panel. But it requires you to have a child page. Tags field will be disabled in the root page editor.

Try to add a new page in the `.\lot\page\article` folder.
